core/src/wallet/ethereum/EthereumLikeWallet.cpp

```diff
@@ -64,6 +64,7 @@ namespace ledger {
             _observer = observer;
             _keychainFactory = keychainFactory;
             _synchronizerFactory = synchronizer;
+            _coinType = scheme.getCoinType() ? scheme.getCoinType() : network.bip44CoinType;
```

Comment:
Ok, I was not look at this close enought, I was hopping to path coinType as a constructor argument.
But probably it will envolve more changes...

Reply:
Exactly !